salt/modules/icinga2.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide icinga2 compatibility to salt.
+
+:depends:   - icinga2 server
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+import os
+import subprocess
+
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    '''
+    Only load this module if the mysql libraries exist
+    '''
+    # TODO: icinga2 check should be os independent and maybe based on package, not binary.
+    if os.path.isfile('/usr/sbin/icinga2'):
+        return True
+    return (False, 'Icinga2 not installed.')
+
+
+def _execute(cmd, ret_code=False):
```

Comment:
Is there a reason we're doing this instead of simply cross-calling to Salt's `cmd.run`?